hack/tools/install_protoc.sh

```diff
@@ -17,15 +17,14 @@ case ${os} in
         ;;
 esac
 
-version=3.7.1
 
-header_text "Checking for bin/protoc"
-[[ -f bin/protoc ]] && exit 0
+header_text "Checking for bin/protoc ${PROTOC_VERSION}"
+[[ -f bin/protoc && `bin/protoc --version | awk '{print $2}'` == ${PROTOC_VERSION} ]] && exit 0
 
 header_text "Installing bin/protoc"
```

Comment:
I'd add the version number too

Reply:
Don't bother. It is going to be removed.